code/src/index.css

```diff
@@ -5,9 +5,71 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: mistyrose;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.app-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 20px;
+}
+
+/*FORM STYLING*/
+
+form {
+  max-width: 350px;
+  border: solid black 1px;
+  font-family: 'Inconsolata', monospace;
+  box-shadow: 6px 6px black;
+  background-color: #f2f0f0;
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  margin-top: 30px;
+  margin-bottom: 15px;
+}
+
+.send-button {
+  font-family: 'Inconsolata', monospace;
+  border-radius: 30px;
+  border: none;
+  padding: 10px;
+  margin: 10px;
+  background-color: #ffadad;
+}
+
+.message-length {
+  float: right;
+}
+
+/*THOUGHTS LIST AND LIKE BUTTON STYLING*/
+
+.message {
+  font-family: 'Inconsolata', monospace;
+  border: solid black 1px;
+  box-shadow: 6px 6px black;
+  width: 350px;
+  padding: 10px;
+  margin-bottom: 30px;
+  background-color: #fff;
+}
```

Comment:
consider adding ` word-break: break-all;` to fix instances when text flows out of the box: 
<img width="615" alt="Screenshot 2020-11-04 at 3 16 41 PM" src="https://user-images.githubusercontent.com/64912576/98122505-fd703a00-1eb0-11eb-9c11-cfde03f291bb.png">